tachyon/zk/plonk/layout/floor_planner/v1/v1_floor_planner.h

```diff
@@ -47,7 +47,7 @@ class V1FloorPlanner : public FloorPlanner<Circuit> {
     MeasurementPass<F> measure;
     {
       V1Pass<F> pass(&measure);
-      circuit.WithoutWitness()->Synthesize(std::move(config), &pass);
+      circuit.WithoutWitness()->Synthesize(std::move(config.Clone()), &pass);
```

Comment:
Since `config.Clone()` is prvalue in https://en.cppreference.com/w/cpp/language/value_category, you don't need to do `std::move()`. 

```suggestion
      circuit.WithoutWitness()->Synthesize(config.Clone(), &pass);
```